common/test/test_lint.py

```diff
@@ -37,43 +119,106 @@ def _collect_py_files(self) -> Iterable[pathlib.Path]:
         path = path.parent
 
         # Find recursive all py-files.
-        return path.rglob('**/*.py')
+        py_files = path.rglob('**/*.py')
```

Comment:
I think this should either be:

```suggestion
        py_files = path.glob('**/*.py')
```

OR

```suggestion
        py_files = path.rglob('*.py')
```

See the docs for [rglob](https://docs.python.org/3/library/pathlib.html#pathlib.Path.rglob), which state:

> This is like calling [Path.glob()](https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob) with "`**/`" added in front of the pattern